pkg/util/partition_message_test.go

```diff
@@ -99,12 +87,20 @@ func testMessageItemsFromLines(hundreds int, limits types.MessageLimit, repeat i
 		}
 	}
 
-	items, omitted = MessageItemsFromLines(builder.String(), limits)
+	batches = MessageItemsFromLines(builder.String(), limits)
+
+	// maxChunkSize := Min(limits.ChunkSize, repeat*100)
+
+	// totalChunks := 0
+	// for _, batchChunks := range batches {
+	// 	totalChunks += len(batchChunks)
+	// }
 
-	maxChunkSize := Min(limits.ChunkSize, repeat*100)
+	//expectedTotalChunkCount := hundreds / repeat
+	//expectedChunkCountInFirstBatch := Min(limits.TotalChunkSize/maxChunkSize, Min(expectedTotalChunkCount, limits.ChunkCount))
+	// Expect(len(batches[0])).To(Equal(expectedChunkCountInFirstBatch), "Chunk count in first batch")
 
-	expectedChunkCount := Min(limits.TotalChunkSize/maxChunkSize, Min(hundreds/repeat, limits.ChunkCount-1))
-	Expect(len(items)).To(Equal(expectedChunkCount), "Chunk count")
+	// Expect(totalChunks).To(Equal(expectedTotalChunkCount), "Total chunk count")
```

Comment:
Please remove or uncomment before merging.

Reply:
Yep. This was a test that doesn't make any sense anymore, but still needs to be replaced.